app/c/[slugKey]/nase_spominske/page.js

```
@@ -2,28 +2,54 @@
+  const [monthlyTotalData, setMonthlyTotalData] = useState({});
+  const [currentMonthData, setCurrentMonthlData] = useState(null);
+
+  const getMonthlyData = async () => {
+    try {
+      const response = await obituaryService.getMonthlyCompanyData();
+      if (response) {
+        setMonthlyTotalData(response.obituaries);
+        setTotalObitWithKeeper(response.totalObituariesWithKeeper);
+        const now = new Date();
+        const formatted = now.toLocaleString('en-US', { month: 'long', year: 'numeric' });
+        if (typeof response.obituaries[formatted] !== undefined) {
```

Biome (2.1.2)

[error] Invalid typeof comparison.

Compare with one of the following string literals:


(lint/correctness/useValidTypeof)